client/my-sites/plugins/plugins-browser-item/index.jsx

```diff
@@ -16,137 +19,180 @@ import './style.scss';
+			: []
+	);
+
+	const dateFromNow = useMemo(
+		() =>
+			plugin.last_updated ? moment.utc( plugin.last_updated, 'YYYY-MM-DD hh:mma' ).fromNow() : null,
+		[ plugin.last_updated ]
+	);
+
+	const pluginLink = useMemo( () => {
+		if ( plugin.link ) {
+			return plugin.link;
 		}
 
-		let url = '/plugins/' + this.props.plugin.slug;
-		if ( this.props.site ) {
-			url += '/' + this.props.site;
+		let url = '/plugins/' + plugin.slug;
+		if ( site ) {
+			url += '/' + site;
 		}
 		return url;
-	}
+	}, [ plugin ] );
 
-	trackPluginLinkClick = () => {
+	const trackPluginLinkClick = useCallback( () => {
 		recordTracksEvent( 'calypso_plugin_browser_item_click', {
-			site: this.props.site,
-			plugin: this.props.plugin.slug,
-			list_name: this.props.listName,
+			site: site,
+			plugin: plugin.slug,
+			list_name: props.listName,
 		} );
-	};
+	}, [ site, plugin, props.listName ] );
 
-	isWpcomPreinstalled() {
-		if ( this.props.plugin.isPreinstalled ) {
+	const isWpcomPreinstalled = useMemo( () => {
+		if ( plugin.isPreinstalled ) {
 			return true;
 		}
 
-		if ( ! this.props.site ) {
+		if ( ! site ) {
 			return false;
 		}
 
-		return ! this.props.isJetpackSite && includes( PREINSTALLED_PLUGINS, this.props.plugin.name );
-	}
+		return ! isJetpack && includes( PREINSTALLED_PLUGINS, plugin.name );
+	}, [ isJetpack, site, plugin ] );
 
-	renderInstalledInOrPricing() {
-		const { sitesWithPlugin, translate } = this.props;
-		if ( ( sitesWithPlugin && sitesWithPlugin.length > 0 ) || this.isWpcomPreinstalled() ) {
-			return (
-				/* eslint-disable wpcalypso/jsx-gridicon-size */
-				<div className="plugins-browser-item__installed">
-					<Gridicon icon="checkmark" size={ 14 } />
-					{ this.props.translate( 'Installed' ) }
-				</div>
-				/* eslint-enable wpcalypso/jsx-gridicon-size */
-			);
-		}
-
-		return <div className="plugins-browser-item__pricing">{ translate( 'Free' ) }</div>;
+	if ( isPlaceholder ) {
+		return <Placeholder iconSize={ iconSize } />;
 	}
 
-	renderUpgradeButton() {
-		const { isPreinstalled, upgradeLink } = this.props.plugin;
-		if ( isPreinstalled || ! upgradeLink ) {
-			return null;
-		}
+	const classNames = classnames( 'plugins-browser-item', variant );
+	return (
+		<li className={ classNames }>
+			<a
+				href={ pluginLink }
+				className="plugins-browser-item__link"
+				onClick={ trackPluginLinkClick }
+			>
+				<div className="plugins-browser-item__info">
+					<PluginIcon size={ iconSize } image={ plugin.icon } isPlaceholder={ isPlaceholder } />
+					<div className="plugins-browser-item__title">{ plugin.name }</div>
+					{ variant === PluginsBrowserElementVariant.Extended && (
+						<>
+							<div className="plugins-browser-item__author">
+								{ translate( 'by ' ) }
+								<span className="plugins-browser-item__author-name">{ plugin.author_name }</span>
+							</div>
+							{ dateFromNow && (
+								<div className="plugins-browser-item__last-updated">
+									{ translate( 'Last updated ' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **144** times:
`translate( 'Last Updated:' )` *ES Score: 9*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/enhance-plugins-browser-item)